standards-in-genomic-sciences.csl

```diff
@@ -182,7 +182,7 @@
           <text macro="editor" prefix=" " suffix="."/>
           <group prefix=" " suffix=".">
             <text variable="container-title" font-style="italic" form="short" suffix=" "/>
-            <group delimiter=";" prefix=" ">
+            <group delimiter=";">
               <date variable="issued">
                 <date-part name="year"/>
               </date>
```

Comment:
Yeah, I had the same thought. But this can become more tricky if the only child node has children itself. Moreover, AFAIK one can use `group` also for a conditional existence statement.